.github/workflows/linter.yml

```diff
@@ -0,0 +1,27 @@
+name: Linter
+
+on: [push, pull_request]
+
+jobs:
+  linter:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Cache NPM dependencies
+        uses: actions/cache@v1
+        with:
+          path: node_modules
+          key: ${{ runner.OS }}-npm-cache
+          restore-keys: |
+            ${{ runner.OS }}-npm-cache
+      - name: Install Dependencies
+        run: yarn --frozen-lockfile --non-interactive
```

Comment:
Although `yarn` should be considered as faster, we prefer using `npm` in CI (Like what we do in Travis)